src/components/Plants/DashboardChart.tsx

```diff
@@ -35,6 +35,7 @@ export default function DashboardChart(props: DashboardChartProps): JSX.Element
               data: chartValues,
               barThickness: 50, // number (pixels) or 'flex'
               backgroundColor: colors,
+              minBarLength: 3,
```

Comment:
that was it? nice :)

Reply:
Yes :(  after fighting a lot with the code, i realize there was this prop :(

Reply:
works in vercel 🎉 👏